include/psmove.h

```diff
@@ -194,6 +231,18 @@ enum PSMove_Version {
      **/
     PSMOVE_CURRENT_VERSION = 0x030001, /*!< Current version, see psmove_init() */
 };
+    
+/* Swig complains that the following variable declarations may leak memory. */
+
+ADDAPI extern const PSMove_3AxisTransform *k_psmove_zero_transform;
```

Comment:
Can those be put into `psmove_private.h`? If not, maybe we should have API methods for getting/setting those values instead of `extern` declarations.